spec/default_facts.yml

```diff
@@ -0,0 +1,8 @@
+# Use default_module_facts.yml for module specific facts.
```

Comment:
I don't think we need this file. @HelenCampbell do you know if we need this?

Reply:
Nah this can be removed pretty sure.